rpcs3/util/simd.hpp

```diff
@@ -3107,6 +3135,159 @@ inline auto gv_shuffle_right(A&& a)
 	FOR_X64(unary_op, kIdPsrldq, kIdVpsrldq, std::forward<A>(a), Count);
 }
 
+// Load 16 bytes of aligned integer data
+inline v128 gv_load(const void* ptr)
+{
+#if defined(ARCH_X64)
+	return _mm_load_si128(static_cast<const __m128i*>(ptr));
+#elif defined(ARCH_ARM64)
+	return vld1q_s32(static_cast<const s32*>(ptr));
+#endif
+}
+
+// Load 16 bytes of aligned floating point data
+inline v128 gv_loadfs(const void* ptr)
+{
+#if defined(ARCH_X64)
+	return _mm_load_ps(static_cast<const f32*>(ptr));
```

Comment:
There is no significant performance difference between them in modern CPUs, hence why MSVC doed not emit it. Nor should we as the risks outweighs the advantage.

```suggestion
	return _mm_loadu_ps(static_cast<const f32*>(ptr));
```